pkg/convert/python/python.go

```diff
@@ -259,7 +268,7 @@ func (c *PythonContext) generateEnvironment(pack Package) apkotypes.ImageConfigu
 		"wolfi-base",
 		"busybox",
 		"build-base",
-		"python-" + c.PythonVersion,            //Set the python version requested
+		normalizePythonVersion(c.PythonVersion),
```

Comment:
```suggestion
		"python" + c.PythonVersion,
```

Reply:
Don't we want python3 or python-3.11 here since the normalize bakes in the python? Or maybe I'm missing something here?